src/AnnotationDialog.js

```diff
@@ -331,15 +332,11 @@ class AnnotationDialog extends EventEmitter {
      */
     addSortedAnnotations(annotations) {
```

Comment:
Since this is doing a complete overwrite of existing annotations, can we rename this function?